Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.12-alpine
+
+LABEL maintainer="Karthic Rao<karthic@dgraph.io>"
+
+ENV GOPATH /go
```

Comment:
This shouldn't be required, golang-alpine image should already have GOPATH set.

Reply:
Removing it.